be/src/storage/CMakeLists.txt

```diff
@@ -153,4 +153,5 @@ add_library(Storage STATIC
     compaction_manager.cpp
     compaction_scheduler.cpp
     compaction_task_factory.cpp
+    base_cumulative_compaction_policy.cpp
```

Comment:
1. base_and_cmulative_compaction_policy.cpp
2. The commit message should be changed also.